### PR TITLE
Summarizing some codes

### PR DESCRIPTION
**Pull request by aibon78**
Making some codes shorter

### CODE REVIEW

File: docs/docs/examples/url-shortener.md [hunk 1]
```diff
@@ -187,14 +187,14 @@ class ShortenerConfig:
...
             environment={
-                "URLS_TABLE_NAME": services.dynamo_db.urls_table.table_name,
+                "URLS_TABLE_NAME": services.dynamodb.urls_table.table_name,
                 "BASE_URL": context.resources["base_url"],
             },
         )
 
         services.api_gateway.create_endpoint("POST", "/urls", function, public=True)
 
-        services.dynamo_db.urls_table.grant_write_data(function)
```
Comment (GuiPimenta-Dev) [inline docs/docs/examples/url-shortener.md:197]: We need to alter here to use the method from the dynamodb class as well

File: docs/docs/examples/url-shortener.md [hunk 2]
```diff
@@ -32,17 +32,17 @@ Next, open your project’s `cdk.json` file and incorporate the tables ARNs into
 
 Next, we need to create a new variable class within the DynamoDB class to reference our URLs tables.
```
Comment (GuiPimenta-Dev) [inline docs/docs/examples/url-shortener.md:34]: Lets also put the diagram

File: docs/docs/examples/url-shortener.md (hunk 1 above)
Comment (aibon78) [reply docs/docs/examples/url-shortener.md:197]: I will change

File: docs/docs/examples/url-shortener.md (hunk 2 above)
Comment (aibon78) [reply docs/docs/examples/url-shortener.md:34]: ok

**Pull request closed without merging**